### DISCUSSION TITLE
Catcher has screen tearing during movement while using Metal

### DISCUSSION THREAD

**Discussion by theredpro-sj** [ppy/osu#30432] [Q&A]
### Type

Game behaviour

### Bug description

There seems to be glitchyness/screen tearing whenever the catcher moves while using the Metal renderer. It seems to occur on every skin regardless if it was made for laser or classic. I tested with different renderers and it happens on both the Metal and Metal (experimental) renderers. I am unsure of whether this issue is only prevalent on Mac OS or not However whenever I tried to get a screenshot or video of the issue it did not show up, thus the lack of screenshots.

### Version

2024.1009.1-lazer

### Logs

[compressed-logs.zip](https://github.com/user-attachments/files/17532008/compressed-logs.zip)

**Comment by peppy** [comment]
Can you switch to single thread mode and make sure your frame limiter to vsync
